src/main/java/com/google/sps/data/ItemFeedbackService.java

```diff
@@ -0,0 +1,80 @@
+package com.google.sps.data;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.FetchOptions;
+import com.google.appengine.api.datastore.Key;
+import com.google.appengine.api.datastore.Query;
+
+public class ItemFeedbackService {
+
+    private static final String ITEM_FEEDBACK = "itemFeedback";
+
+    private final DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+
+    public List<ItemFeedback> load(long itemId) {
```

Comment:
itemId should be `learningPathId`. To make this clearer, the method name could be something like `findByLearningPath(..)`.